root/src/main/java/com/example/servingwebcontent/ConnectDatabase.java

```diff
@@ -15,42 +15,46 @@ public ConnectDatabase (String database, String user, String pass) {
 		}
 	}
 
-	public ResultSet getMovie (String id) {
+	public Movie getMovie (String id) {
 		ResultSet rs = null;
-		String query = "select * from movies where id=?";
+		String query = "select * from movies where " + Movie.getColumns()[0] + " = ?";
+		Movie movie = null;
 		try {
 			PreparedStatement stmt = con.prepareStatement(query);
 			stmt.setString(1, id);
 			rs = stmt.executeQuery();
+			if (rs.next()) {
+				movie = new Movie(rs.getString(Movie.getColumns()[2]), rs.getString(Movie.getColumns()[1]), rs.getString(Movie.getColumns()[2]), rs.getString(Movie.getColumns()[3]), rs.getInt(Movie.getColumns()[4]), rs.getString(Movie.getColumns()[5]));
```

Comment:
this isn't really a great way of doing it since you can't really tell what index corresponds to which column name. I would have to dig into getColumns() to tell what 0, 1, 2, etc correspond to. What I meant was to just use public static final strings as constants. Or, if you're comfortable, you can also try using an enum.